### PR TITLE
feat(risk): update score for convex factory, add st-ycrv and sturdy group

### PR DESCRIPTION
**Pull request by kyhoon**
Handles the latest risk framework issues.

For the Gen Lender Sturdy group, I have added "sturdy" in the exclude criteria for the strategy lender groups in the Mainnet and Optimism.

### CODE REVIEW

File: data/risks/networks/1/strategylender.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
 			"genericlender"
 		],
 		"strategies": [],
-		"exclude": []
```
Comment (storming0x) [inline data/risks/networks/1/strategylender.json:16]: i think this one needs to exclude the address 0x5F84acB4a27D762FC0E5CD6EFc1674827c0934C4 also

File: data/risks/networks/10/strategylender.json [hunk 2]
```diff
@@ -13,6 +13,8 @@
 			"genericlender"
 		],
 		"strategies": [],
-		"exclude": []
+		"exclude": [
```
Comment (storming0x) [inline data/risks/networks/10/strategylender.json:16]: this one is needed only for mainnet at the moment

Comment (kyhoon) [reply data/risks/networks/10/strategylender.json:16]: alright, will leave the file in optimism

**Review by rareweasel — DISMISSED** [review]
StYCrv LGTM

**Comment by rareweasel** [comment]
@kyhoon any news on this?

File: data/risks/networks/1/strategylender.json (hunk 1 above)
Comment (kyhoon) [reply data/risks/networks/1/strategylender.json:16]: added in b12932f

File: internal/strategies/risk.framework.go [hunk 3]
```diff
@@ -144,6 +144,9 @@ func getStrategyGroup(chainID uint64, strategy *TStrategy) *TStrategyGroupFromRi
 		if helpers.ContainsSubString(toLowerExclude, toLowerName) {
```
Comment (rareweasel) [inline internal/strategies/risk.framework.go:144]: Great the exclude field works with names.

**Review by rareweasel — APPROVED** [review]
LGTM yCRV scores

**Review by rareweasel — APPROVED** [review]
yCRV LGTM

**Pull request merged**